samples/snippets/pom.xml

```diff
@@ -64,7 +64,7 @@
     <dependency>
       <groupId>com.google.cloud</groupId>
       <artifactId>google-cloud-pubsublite</artifactId>
-      <version>0.0.1-SNAPSHOT</version><!--    TODO: remove after release    -->
+      <version>0.1.4</version><!-- {x-version-update:google-cloud-pubsublite:current} -->
```

Comment:
This will be removed later because libraries-bom currently doesn't have the latest Pub/Sub Lite java library.